src/header.jl

```diff
@@ -378,17 +377,17 @@ end
 
 # Comments
 """
-    get_comment(hdr, key)
+    get_comment(hdr::FITSHeader, key)
```

Comment:
```suggestion
    get_comment(hdr::FITSHeader, key_or_index::Union{String,Integer})
```
Or maybe
```suggestion
    get_comment(hdr::FITSHeader, key::String)
    get_comment(hdr::FITSHeader, index::Integer)
```
Up to you